src/components/Project/ProjectDashboard.tsx

```diff
@@ -37,7 +36,7 @@ import { failedToLoadExecutionsString } from './constants';
 const useStyles = makeStyles((theme: Theme) => ({
   projectStats: {
     paddingTop: theme.spacing(7),
-    paddingBottom: theme.spacing(7),
+    paddingBottom: theme.spacing(5),
```

Comment:
This will keep spacing below the totals section equal to the top one, accounted for marginTop of the title of the next section no matter which one it is domain settings of bar charts
<img width="1458" alt="Screen Shot 2022-04-08 at 10 54 22 AM" src="https://user-images.githubusercontent.com/101579322/162479246-0d1278fa-4c3a-40d5-ad0d-557036e5d993.png">